projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-external-media.php

```diff
@@ -48,6 +48,21 @@ class WPCOM_REST_API_V2_Endpoint_External_Media extends WP_REST_Controller {
 			'title'   => array(
 				'type' => 'string',
 			),
+			'meta'    => array(
+				'type'                 => 'object',
+				'additionalProperties' => false,
+				'properties'           => array(
+					'vertical_id'   => array(
+						'type' => 'string',
```

Comment:
[It doesn't look like string values are actually sanitized](https://github.com/WordPress/wordpress-develop/blob/b64d06f7840a18f86a42ad14c8bff80fe6fc1456/src/wp-includes/rest-api.php#L2794-L2796) unless [we add a format](https://github.com/WordPress/wordpress-develop/blob/b64d06f7840a18f86a42ad14c8bff80fe6fc1456/src/wp-includes/rest-api.php#L2786-L2787). Should we do that?

Reply:
I added this, but after sending a vertical_id of `test_id_with_spaces_at_end        ` (there are multiple spaces at the end of this that github is filtering out), it's saved that way to the database, which [makes me think it's not working](https://github.com/WordPress/wordpress-develop/blob/b64d06f7840a18f86a42ad14c8bff80fe6fc1456/src/wp-includes/formatting.php#L5458). (Expected to see: Multiple spaces collapsed to one)

Reply:
Maybe _all_ of these sanitations don't work nested?